cpp/tests/io/parquet_test.cpp

```diff
@@ -4215,15 +4215,15 @@ TEST_F(ParquetWriterTest, ByteArrayStats)
   // check that byte array min and max statistics are written as expected. If a byte array is
   // written as a string, max utf8 is 0xf7bfbfbf and so the minimum value will be set to that value
   // instead of a potential minimum higher than that.
-  std::vector<uint8_t> expected_col0_min{0xf0};
-  std::vector<uint8_t> expected_col0_max{0xf0, 0xf5, 0xf5};
-  std::vector<uint8_t> expected_col1_min{0xfe, 0xfe, 0xfe};
-  std::vector<uint8_t> expected_col1_max{0xfe, 0xfe, 0xfe};
-
-  cudf::test::lists_column_wrapper<uint8_t> list_int_col0{
-    {0xf0}, {0xf0, 0xf5, 0xf3}, {0xf0, 0xf5, 0xf5}};
-  cudf::test::lists_column_wrapper<uint8_t> list_int_col1{
-    {0xfe, 0xfe, 0xfe}, {0xfe, 0xfe, 0xfe}, {0xfe, 0xfe, 0xfe}};
+  std::vector<uint8_t> expected_col0_min{0x70};
```

Comment:
Match the type (`int8_t`) here. However, the bigger question is whether this test still fits the criteria explained in the comment. It appears it does not exceed the maximum UTF-8 value anymore.

Reply:
I wouldn't change this test.  Binary in parquet is unsigned, so you still want to test that `0xfe` doesn't wind up as `0xf7bfbfbf` in the statistics if it's written as binary and not string.  This should have nothing to do with how binary is represented internally in cuDF columns.

Reply:
Thank you @etseidl, It is important to remember the origin of these values. They were selected specifically to ensure that the stats were not for a string by being one larger than the max UTF-8 value.

> Match the type (int8_t) here.

I originally was doing that, but as Ed mentions below, binary stats in parquet are unsigned and I am comparing to the actual stats values.